src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/documents/DocumentApiClient.kt

```diff
@@ -30,9 +30,10 @@ class DocumentApiClient(@Qualifier("oauthDocumentApiClient") val documentApiClie
     .accept(MediaType.APPLICATION_JSON)
     .retrieve()
     .onStatus(HttpStatusCode::isError) { response ->
-      val errorResponse = response.bodyToMono<Map<String, String>>()
-      error("Error during uploading document (status=${response.statusCode()}, body=$errorResponse)")
+      response.bodyToMono<Map<String, String>>().map { json ->
```

Comment:
if you're not using the structure of the payload, wondering if it'd make sense to make this less typesafe.. I'm wondering if it would fail to log properly on an nginx 502 error page for instance as it wouldn't be able to parse the html into a map?

Reply:
and thinking jackson might fail to parse the json unless its strictly conforms to a `Map<String, String>`?

(could be wrong!)